x-pack/solutions/observability/plugins/slo/public/pages/slos/components/health_callout/health_callout.tsx

```diff
@@ -83,69 +74,25 @@ export function HealthCallout({ sloList }: { sloList: SLOWithSummaryResponse[] }
           <EuiFlexItem>
             <FormattedMessage
               id="xpack.slo.sloList.healthCallout.description"
-              defaultMessage="The following {count, plural, one {transform is} other {transforms are}
-          } in an unhealthy state:"
+              defaultMessage="The following {count, plural, one {SLO is} other {SLOs are}}
```

Comment:
Do we need to remove the translations if we modify this message? So that they can get recreated.